### ISSUE TITLE
dotnet PKG rejected by vanilla OS X 10.11.1

### ISSUE THREAD

**Issue by piotrpMSFT** [dotnet/cli#657]
The PKG is not able to install on 10.11.1.

````
"dotnet-osx-x64.latest.pkg" can’t be opened because it is from an unidentified developer.

Your security preferences allow installation of only apps from the Mac App Store and identified developers.

Safari downloaded this file today at 1:50 PM from github.com.
````

**Comment by blackdwarf** [comment]
@piotrpMSFT is this from the terminal or GUI? Also, I will take a look at this, my guess we need to somehow sign or something. 

In the meantime, the workaround is to save locally and then right-click and choose "Open". Same warning, but will give the "Install anyway" button (or similar).

**Comment by blackdwarf** [comment]
Yup, the PKG needs to be signed. This will be solved as #500 gets done, so I will close this one.
